src/pdf/ShippingLabels/ShippingLabelsPdfButton.tsx

```diff
@@ -131,14 +133,14 @@ const getRequiredData = async (
             collection_centre: parcel.collection_centre?.acronym ?? "",
             collection_datetime: formatDatetime(parcel.collection_datetime),
             voucher_number: parcel.voucher_number ?? "",
-            full_name: client.full_name,
-            phone_number: client.phone_number,
-            address_1: client.address_1,
-            address_2: client.address_2,
-            address_town: client.address_town,
-            address_county: client.address_county,
-            address_postcode: client.address_postcode,
-            delivery_instructions: client.delivery_instructions,
+            full_name: clientIsActive ? client.full_name ?? "" : displayNameForDeletedClient,
+            phone_number: clientIsActive ? client.phone_number ?? "" : "-",
+            address_1: clientIsActive ? client.address_1 ?? "" : "",
+            address_2: clientIsActive ? client.address_2 ?? "" : "",
+            address_town: clientIsActive ? client.address_town ?? "" : "",
```

Comment:
Is this necessary or would the address be empty for deleted clients and still return ""?

Reply:
Do you mean I should add the comment here, or in seed.mts?

Reply:
Just here please

Reply:
Have done